server/handlers/project_command_output_handler.go

```diff
@@ -0,0 +1,134 @@
+package handlers
+
+import (
+	"sync"
+
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
+)
+
```

Comment:
create a new function here to initialize the `DefaultProjectCommandOutputHandler`,

```
func NewProjectCommandOutputHandler(projectCmdOutput chan *models.ProjectCmdOutputLine, logger logging.SimpleLogging) ProjectCommandOutputHandler {
    return &DefaultProjectCommandOutputHandler{
        ProjectCmdOutput: projectCmdOutput,
        logger: logger,
        controllerBuffer: map[string]map[chan string]bool{},
    }
}
```

And use it in the server.go